_posts/work/2018-01-31-credit-models-with-dask.md

```diff
@@ -158,17 +184,19 @@ This allows us to write each equation as it's own isolated function and mark it'
+This graph was a bit too large to render with the normal `my_task.visualize()` method, so instead we rendered it with [Gephi](https://gephi.org) and make the pretty colored graph above. The chaotic upper region of this graph is the individual equation calculations. Zooming in we can see the entry point, our input pandas DataFrame, as the large orange circle at the top and how it gets fed into many of the original equations.
 
 <img src="{{BASE_PATH}}/images/credit_models/graph_model.png" alt="zoomed model section">
 
 The output of the model is about 100 times the size of the input so we do some aggregation at the end via tree reduction. This accounts for the more structured bottom half of the graph. The large green node at the bottom is our final output.
 
 <img src="{{BASE_PATH}}/images/credit_models/graph_agg.png" alt="zoomed agg section">
 
+
 ## Final Thoughts
 
-With our dask-based data structure, we spend more of our time writing model code rather than maintenance of the engine itself. Dask also offers a number of advantages not covered above. For example, with dask you also get access to [diagnostics](https://distributed.readthedocs.io/en/latest/web.html) such as time spent running each task and resources used. Also, you can easily distribute your computation with [dask distributed](https://distributed.readthedocs.io/en/latest/) with relative ease. Now if I want to run our model across out-of-core distributed data, we don't have to worry about extending our code to incorporate something like Spark. Finally, dask allows you to give pandas-capable business analysts or less technical folks access to large datasets with the [dask dataframe](http://dask.pydata.org/en/latest/dataframe.html).
+With our dask-based data structure, we spend more of our time writing model code rather than maintenance of the engine itself. This allows a clean separation between our analysts that design and write our models, and our computational system, tha runs them.  Dask also offers a number of advantages not covered above. For example, with dask you also get access to [diagnostics](https://distributed.readthedocs.io/en/latest/web.html) such as time spent running each task and resources used. Also, you can easily distribute your computation with [dask distributed](https://distributed.readthedocs.io/en/latest/) with relative ease. Now if I want to run our model across larger-than-memory data or on a distributed cluster, we don't have to worry about rewriting our code to incorporate something like Spark. Finally, dask allows you to give pandas-capable business analysts or less technical folks access to large datasets with the [dask dataframe](http://dask.pydata.org/en/latest/dataframe.html).
```

Comment:
computational system, tha**t** runs them.

Nice cleanup of this paragraph!

Reply:
Fixed